internal-dependencies/pom.xml

```diff
@@ -80,6 +80,7 @@
         <artifactId>servlet-api</artifactId>
         <version>2.5</version>
       </dependency>
+      <!-- Using Jakarta Expression Language 4.0 for Java 8 compatibility -->
```

Comment:
I think this comment (although only copied / moved) might be confusing as we do not support Java 8 anymore

Reply:
I was thinking about it and thought to let it in because 4.0.0 is not the latest release. I would like to upgrade to latest Jakarta EL release with a later issue.

But brings me to the point to just remove now this comment and raise the issue for the upgrade.

Reply:
Issue raised: https://github.com/operaton/operaton/issues/252

Reply:
Comment removed